### ISSUE TITLE
train question

### ISSUE THREAD

**Issue by OhhAlice1995** [generalized-iou/g-darknet#5]
I have a question why when I train yolo, the obj: is smaller and smaller, and around 400 iterations, IoU and GIOU will suddenly become zero. And so do the other scores, such as class and .5R. However, the loss seem to be normal.

**Comment by dishen12** [comment]
I have the same question

**Comment by nathantsoi** [comment]
I had a similar issue when starting out and reading through the now-banned [darknet google group](https://groups.google.com/forum/?nomobile=true#!forum/darknet), some NaNs are expected / OK, but if everything goes to NaN there is a problem. Batch size / learning rate / number of GPUs could all be factors. Did you adjust any of these? Also, are you using the supplied configurations as described in the [pretrained model](https://github.com/generalized-iou/g-darknet#pre-trained-models) documentation or did you build your own config?

By the way, if a googler happens to read this, can you help restore the darknet google group? It was actually really helpful.

**Comment by OhhAlice1995** [comment]
Now the issue has been sovled. Thanks a lot.

**Comment by OhhAlice1995** [comment]
Now the issue has been solved. Thanks a lot.

**Comment by wangzhiming68** [comment]
I have the same question. When I train with my own data (about 8000),  IoU and GIOU will suddenly become zero around 360 iterations. I tried several times with batch=64 and different subdivisions of 16, 8. Single GUP, learning rate =0.001.
However, these data and labels train well on original yolov3.
